tests/Unit/VoyagerTest.php

```diff
@@ -42,6 +42,27 @@ public function testDimmersReturnsCollectionOfConfiguredWidgetsWhichShouldBeDisp
 
         $dimmers = Voyager::dimmers();
 
-        $this->assertEquals(1, $dimmers->count());
+        $this->assertEquals(1, $dimmers[0]->count());
+    }
+
+    /**
+     * Dimmers returns an array filled with widget collections.
+     *
+     * This test ensures that each widget collection has a maximum of three widgets/dimmers
+     * creating as many groups as are needed in order to encompass the full range of dimmers.
+     */
+    public function testEachDimmerGroupHasAMaxAmountOfThreeDimmers()
```

Comment:
If you want to test that Group has max amount of three Dimmers you should set 3 dimmers in config and assert equals 1, then set 4 dimmers and assert equals 2.

Right now you could have 4 dimmers in the first group without error.